geo-replication/syncdaemon/primary.py

```diff
@@ -1536,7 +1536,7 @@ def crawl(self):
         self.changelogs_batch_process(changes)
 
     def register(self, register_time, status):
-        self.sleep_interval = gconf.get("change-interval")
+        self.sleep_interval = gconf.get("change-interval", 60)
```

Comment:
The default value comes from here https://github.com/gluster/glusterfs/blob/9be5e5288f0bea2e000744432f3eb0911ebf01dc/geo-replication/gsyncd.conf.in#L132